contracts/SnappAuction.sol

```diff
@@ -99,8 +98,7 @@ contract SnappAuction is SnappBase {
         bytes32 _currStateRoot,
         bytes32 _newStateRoot,
         bytes32 _orderHash,
-        bytes memory prices,
-        bytes memory volumes
+        bytes memory solution  // represents (bytes prices, bytes volumes)
```

Comment:
nit: you could call it pricesAndVolumes to save that comment.

Reply:
Ya, although @twalth3r keeps insisting that it isn't actually volumes, but "amounts". maybe he can suggest a great name for this. Otherwise I will go with `pricesAndVolumes`